ee/clickhouse/queries/clickhouse_funnel_trends.py

```diff
@@ -0,0 +1,126 @@
+from datetime import datetime, timedelta
+
+from ee.clickhouse.client import sync_execute
+from ee.clickhouse.models.property import parse_prop_clauses
+from ee.clickhouse.queries.clickhouse_funnel_base import ClickhouseFunnelBase
+from ee.clickhouse.queries.util import get_time_diff, get_trunc_func_ch, parse_timestamps
+from ee.clickhouse.sql.events import NULL_SQL_FUNNEL_TRENDS
+from ee.clickhouse.sql.funnels.funnel_trend import FUNNEL_TREND_SQL
+from ee.clickhouse.sql.person import GET_LATEST_PERSON_DISTINCT_ID_SQL
+
+DAY_START = 0
+TOTAL_COMPLETED_FUNNELS = 1
+ALL_FUNNELS_ENTRIES = 2
+PERSON_IDS = 3
+TIMESTAMP_FORMAT = "%Y-%m-%d %H:%M:%S"
+HUMAN_READABLE_TIMESTAMP_FORMAT = "%a. %-d %b"
+
+
+class ClickhouseFunnelTrends(ClickhouseFunnelBase):
+    def run(self):
+        summary = self.perform_query()
+        ui_response = self._get_ui_response(summary)
+        return ui_response
+
+    def perform_query(self):
+        sql = self._configure_sql()
+        results = sync_execute(sql, self.params)
+        summary = self._summarize_data(results)
+        return summary
+
+    def _configure_sql(self):
+        funnel_trend_null_sql = self._get_funnel_trend_null_sql()
+        parsed_date_from, parsed_date_to, _ = self._get_dates()
+        prop_filters, prop_filter_params = self._get_filters()
```

Comment:
can omit `prop_filter_params` on this function return since it's being directly added to self.params now